chroma_core/models/host.py

```diff
@@ -1816,3 +1875,59 @@ def run(self, kwargs):
         self.invoke_rust_agent_expect_result(
             kwargs["host"], "snapshot_unmount", {"fsname": kwargs["fsname"], "name": kwargs["name"]}
         )
+
+
+class MountSnapshotJob(Job):
+    fqdn = models.CharField(max_length=256)
+    fsname = models.CharField(max_length=8)
+    name = models.CharField(max_length=512)
+
+    @classmethod
+    def long_description(cls, stateful_object):
+        return help_text["mount_snapshot"]
+
+    def description(self):
+        return "Mount snapshot on host %s" % self.fqdn
+
+    def get_deps(self):
+        # To prevent circular imports
+        from chroma_core.models.filesystem import ManagedFilesystem
+
+        return DependOn(ManagedFilesystem.objects.get(name=self.fsname), "available")
```

Comment:
We'll want to check if this is needed for ldiskfs since it's apparently not needed with zfs